src/themeMode/ThemeMode.tsx

```diff
@@ -0,0 +1,13 @@
+import { THEME_MODE_KEY } from '../constants/storage';
+
+export const getIsDarkMode = () => {
+  return JSON.parse(window.localStorage.getItem(THEME_MODE_KEY) || 'false');
+};
+
+export const setDarkMode = () => {
+  return window.localStorage.setItem(THEME_MODE_KEY, JSON.stringify(true));
+};
+
+export const removeDarkMode = () => {
+  window.localStorage.setItem(THEME_MODE_KEY, JSON.stringify(false));
+};
```

Comment:
these 2 methods are redundand
You can create only 1 - toggleTheme() - which will change dark if white, and white if current is dark